source/MapDetailPanel.cpp

```diff
@@ -488,52 +523,75 @@ void MapDetailPanel::DrawInfo()
-					uiPoint + Point(-60., -12.),
-					planet->HasShipyard() ? medium : faint);
-				if(commodity == SHOW_SHIPYARD)
-					PointerShader::Draw(uiPoint + Point(-60., -5.), Point(1., 0.),
-						10.f, 10.f, 0.f, medium);
-				
-				font.Draw("Outfitter",
-					uiPoint + Point(-60., 8.),
-					planet->HasOutfitter() ? medium : faint);
-				if(commodity == SHOW_OUTFITTER)
-					PointerShader::Draw(uiPoint + Point(-60., 15.), Point(1., 0.),
-						10.f, 10.f, 0.f, medium);
 				
-				bool hasVisited = player.HasVisited(*planet);
-				font.Draw(hasVisited ? "(has been visited)" : "(not yet visited)",
-					uiPoint + Point(-70., 28.),
-					dim);
-				if(commodity == SHOW_VISITED)
-					PointerShader::Draw(uiPoint + Point(-70., 35.), Point(1., 0.),
-						10.f, 10.f, 0.f, medium);
-				
-				uiPoint.Y() += 130.;
+				// Makes sure one more planet does not go out of the screen.
+				if(uiPoint.Y() <= (Screen::Bottom() - 230 - 130) && currentPlanet >= firstPlanet)
+				{
+					shown.insert(planet);
+					
+					bool hasSpaceport = planet->HasSpaceport();
+					
+					SpriteShader::Draw(planetSprite, uiPoint);
+					planetY[planet] = uiPoint.Y() - 60;
+					
+					font.Draw({object.Name(), alignLeft},
+						uiPoint + Point(-70., -52.),
+						planet == selectedPlanet ? medium : dim);
+					
+					string reputationLabel = !hasSpaceport ? "No Spaceport" :
+						GameData::GetPolitics().HasDominated(planet) ? "Dominated" :
+						planet->GetGovernment()->IsEnemy() ? "Hostile" :
+						planet->CanLand() ? "Friendly" : "Restricted";
+					font.Draw(reputationLabel,
+						uiPoint + Point(-60., -32.),
+						hasSpaceport ? medium : faint);
+					if(commodity == SHOW_REPUTATION)
+						PointerShader::Draw(uiPoint + Point(-60., -25.), Point(1., 0.),
+							10.f, 10.f, 0.f, medium);
+					
+					font.Draw("Shipyard",
+						uiPoint + Point(-60., -12.),
+						planet->HasShipyard() ? medium : faint);
+					if(commodity == SHOW_SHIPYARD)
+						PointerShader::Draw(uiPoint + Point(-60., -5.), Point(1., 0.),
+							10.f, 10.f, 0.f, medium);
+					
+					font.Draw("Outfitter",
+						uiPoint + Point(-60., 8.),
+						planet->HasOutfitter() ? medium : faint);
+					if(commodity == SHOW_OUTFITTER)
+						PointerShader::Draw(uiPoint + Point(-60., 15.), Point(1., 0.),
+							10.f, 10.f, 0.f, medium);
+							
+					bool hasVisited = player.HasVisited(*planet);
+					font.Draw(hasVisited ? "(has been visited)" : "(not yet visited)",
+						uiPoint + Point(-70., 28.),
+						dim);
+					if(commodity == SHOW_VISITED)
+						PointerShader::Draw(uiPoint + Point(-70., 35), Point(1., 0.),
+							10.f, 10.f, 0.f, medium);
+					
+					uiPoint.Y() += 130.;
+				}
+				else
+					++excessPlanet;
+				++currentPlanet;
 			}
 	}
+	// Hints that more planets can be seen by scrolling up.
+	if(excessPlanet > firstPlanet)
+	{
+		const Sprite *up = SpriteSet::Get("ui/up");
+		Point point(Screen::Left() + 98., governmentY + 35.);
+		SpriteShader::Draw(up, point);
```

Comment:
ooooh I didnt know about that I'll check if it looks right
will close the high dpi pr too if it works

Reply:
![pointers](https://user-images.githubusercontent.com/94366726/143732880-c2720ea4-b26c-4fa7-a564-0ae811a9539e.png)
none of them have quite the same look, I could settle for good enough I guess but I would prefer changing all scrolling indicators to this one instead of forgeting it (or just leaving them here)

Reply:
I used them even if they are worse, watever